packages/components/src/globals/scss/_theme-tokens.scss

```
@@ -358,3 +359,41 @@ $tab-underline-disabled: 3px solid $disabled-01 !default;
 /// @access public
 /// @group skeleton
 $skeleton: $skeleton-01 !default; // TODO: Remove this in next major release
+
+// Light UI
+
+// Determine the "light" color for a given token.
+/// @param {String} $token The theme token.
+/// @return {String} The new color value as a hex code or a custom property.
+/// @access public
+/// @group light-ui
+@function get-light-value($token) {
+  $value: map-get($carbon--theme, $token);
+
+  @if $carbon--theme ==
+    $carbon--theme--white or
+    $carbon--theme ==
+    $carbon--theme--g10
+  {
+    $value: darken($value, 8%);
+  }
+
+  @if $carbon--theme ==
+    $carbon--theme--g90 or
+    $carbon--theme ==
+    $carbon--theme--g100
+  {
+    $value: lighten($value, 8%);
+  }
+
+  @if map-get($feature-flags, enable-css-custom-properties) == true {
+    @return var(--cds-#{$token}-light, $value);
```

Would it be worth trying to figure out a function variant of https://github.com/carbon-design-system/carbon/blob/master/packages/themes/scss/_mixins.scss#L14 ?

If not, might be able to use `$custom-property-prefix` like

```suggestion
    @return var(--#{$custom-property-prefix}-#{$token}-light, $value);
```